deployment/travel-advisor.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: travel-advisor
+  labels:
+    name: travel-advisor
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: travel-advisor
+  namespace: travel-advisor
+spec:
+  selector:
+    matchLabels:
+      name: travel-advisor
+  template:
+    metadata:
+      labels:
+        name: travel-advisor
+    spec:
+      containers:
+        - name: travel-advisor
+          image: thisthatdc/travel-advisor:v0.1.3
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          env:
+            - name: OTEL_ENDPOINT
+              value: "https://xbw95514.dev.dynatracelabs.com/api/v2/otlp"
+            - name: OLLAMA_ENDPOINT
+              value: "http://ollama.ollama"
+            - name: TRACELOOP_TELEMETRY
+              value: "false"
+          imagePullPolicy: Always
+          volumeMounts:
+            - name: secrets
+              readOnly: true
+              mountPath: "/etc/secrets"
+      volumes:
+        - name: secrets
+          projected:
+            sources:
+              - secret:
+                  name: dynatrace
+              - secret:
+                  name: pinecone
```

Comment:
to make it work with codespace, we would need two K8s secrets in the ns `travel-advisor` :
1. `dynatrace` with the DT API Key in a `token` field
2. `pinecone` with the PineCone key in a `api-key` field